src/MongoDB/Server.c

```diff
@@ -236,11 +252,13 @@ PHP_METHOD(Server, getLatency)
 		return;
 	}
 
-	if ((sd = mongoc_topology_description_server_by_id(&intern->client->topology->description, intern->server_id, &error))) {
-		RETURN_LONG(sd->round_trip_time);
+	if ((sd = mongoc_client_get_server_description(intern->client, intern->server_id))) {
+		RETVAL_LONG((phongo_long) mongoc_server_description_round_trip_time(sd));
```

Comment:
`mongoc_server_description_round_trip_time()` returns an `int64_t`. It's unlikely that a round trip time would overflow a 32-bit integer, but are we OK with a simple cast for that edge case? This also applies to the Server's debug handler.